docs/src/manifolds/vector_bundle.md

```diff
@@ -1,7 +1,7 @@
 # Vector bundles
 
-Vector bundle $E$ is a manifold that is built on top of another manifold $M$ (base space).
-It is characterized by a continuous function $\Pi \colon E \to M$, such that for each point $x \in M$ the preimage of $x$ by $\Pi$, $\Pi^{-1}(\{x\})$, has a structure of a vector space.
+Vector bundle $E$ is a manifold that is built on top of another manifold $ℳ$ (base space).
+It is characterized by a continuous function $Π : E → ℳ$, such that for each point $x  ∈ ℳ$ the preimage of $x$ by $Π$, $Π^{-1}(\{x\})$, has a structure of a vector space.
```

Comment:
```suggestion
It is characterized by a continuous function $Π : E → ℳ$, such that for each point $p  ∈ ℳ$ the preimage of $p$ by $Π$, $Π^{-1}(\{p\})$, has a structure of a vector space.
```

Reply:
I haven't gotten to the vector-bundle yet with renaming, I am in `E` as Euclidean – but I am also carefully checking writing and grammar.

Reply:
OK, sorry for being too hasty about that.

Reply:
Nevermind. Unifying is a little manual work currently – so it's also good to know what to also look for. I had the `\xi` for SPDs already in mind (I might just `a` instead), but not yet this one.